mtv_dl.py

```diff
@@ -145,17 +145,15 @@
 import urllib.parse
 from pathlib import Path
 from contextlib import contextmanager
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 from functools import partial
-from itertools import chain
-from itertools import islice
+from itertools import chain, islice
```

Comment:
Please import only one module per line (same for timezone). This to simplify merges.